wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
@mikekaganski So the last character in string is now forced to 0 in the last commit I added.

The idea of this global buffer is to allocate the resources only once in the Julia side.
I don't want to allocate 255 bites at each call of a function that may return an error, that would have a really bad impact on functions like `AbstractState_update`.